WNPRC_Purchasing/src/client/components/LineItemsPanel.tsx

```diff
@@ -50,67 +53,94 @@ export const LineItemsPanel: FC<Props> = memo(props => {
     };
 
     return (
-        <Panel
-            className="panel panel-default"
-            expanded={true}
-            onToggle={function () {}} // this is added to suppress JS warning about providing an expanded prop without onToggle
-        >
-            <div className="bg-primary">
-                <Panel.Heading>
-                    <div className="panel-title">
-                        {(!hasRequestId && 'Specify Items') || (hasRequestId && 'Requested Items')}
+        <>
+            {
```

Comment:
{ is only necessary when you're mixing JS into JSX (TS into TSX).  You shouldn't need this here.

Reply:
removed